src/Options.php

```diff
@@ -597,6 +597,28 @@ public function setDefaultIntegrations(bool $enable): void
         $this->options = $this->resolver->resolve($options);
     }
 
+    /**
+     * Gets the truncation length for values in the event payload.
+     *
+     * @return int
+     */
+    public function getTruncationLength(): int
+    {
+        return $this->options['truncation_length'];
+    }
+
+    /**
+     * Sets the truncation length for specific values in the event payload.
```

Comment:
Does this really apply to only some values? I thought it was applied to everything that is processed by the serializers

Reply:
I quickly went over the code, we do not use this value for everything.
Maybe we should tackle this in a different PR.